raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -641,19 +641,43 @@ void assertSentDescribeQuorumResponse(
         long highWatermark,
         List<ReplicaState> voterStates,
         List<ReplicaState> observerStates
+    ) {
+        short apiVersion = (short) (kip853Rpc ? 2 : 1);
```

Comment:
Yep, those take care of supplying the latest versions supported for describeQuorum. I wanted to add functionality to create older versions of describeQuorumResponse - in order to do that I needed to add an apiVersions param to one of these overloaded helpers and alter this one to pass in the apiVersion explicitly (which duplicates what describeQuorumRpcVersion() does). 

I'll at least use describeQuorumRpcVersion() here instead of duplicating the code